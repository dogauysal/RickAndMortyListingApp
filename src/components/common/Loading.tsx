import { HStack, Spinner } from "native-base";
import React from "react";

const Loading = () => {

    return (
        <HStack height={"100%"} space={8} justifyContent="center" alignItems="center">
            <Spinner size="lg" />
        </HStack>
    )
}

export default Loading;