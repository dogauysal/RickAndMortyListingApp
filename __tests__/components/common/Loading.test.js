import { NativeBaseProvider } from "native-base";
import React from "react";
import renderer from "react-test-renderer";
import Loading from "../../../src/components/common/Loading";

describe("Loading Component", () => {
    test("Loading renders correctly", () => {

        const loading = renderer.create(<NativeBaseProvider><Loading /></NativeBaseProvider>).toJSON();
        expect(loading).toMatchSnapshot()
    })
})