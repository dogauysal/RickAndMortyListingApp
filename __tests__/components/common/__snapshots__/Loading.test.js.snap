// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component Loading renders correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
